### DISCUSSION TITLE
求助怎么做增量发布

### DISCUSSION THREAD

**Discussion by fairyspace** [hexojs/hexo#5431] [Question / Answer]
现在情况是这样，由于文章太多，每次hexo g都要花很久，以及考虑到本地磁盘空间有限。需求场景：我想生成并发布A文章，然后本地删除A文章，释放空间，再增加写B文章，生成发布B文章----但是我想线上网站同时拥有A文章和B文章。第二个需求就是第一个需求基础上我已经发布了ABCD文章，但是我想自主删除D文章，其他文章不变。此时我本地没有任何保存的文章，我怎么做到这个增量删除

**Comment by Korltex** [comment]
`hexo g`后面添加参数是可以生成指定目标文件夹的静态网页的，但我不太清楚当`push`至托管平台后，会不会覆盖平台的保存内容
